snowflake/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -0,0 +1,13 @@
+black~=22.0
+flake8>=3.7.9, <4.0
+ipython>=7.31.1, <8.0
+isort~=5.0
+jupyter~=1.0
+jupyterlab~=3.0
+kedro~={{ cookiecutter.kedro_version }}
+kedro-datasets[spark.SparkDataSet]
+kedro-telemetry~=0.2.0
+nbstripout~=0.4
+pytest-cov~=3.0
+pytest-mock>=1.7.1, <2.0
+pytest~=6.2
```

Comment:
Do we need to add snowpark dependency to the file?

Reply:
I'll double check - I think it could be covered from the main kedro setup.py extras_require